Package not building

As of today this package is not building. I realized it when pushing code to a PR that uses your package. Travis output below.
```
kaizen$ wwtd
Ignoring: go, language, notifications
START
./check.sh
github.com/nlopes/slack (download)
github.com/nlopes/slack
# github.com/nlopes/slack
../../../../nlopes/slack/slack.go:104:9: logger.Output undefined (type stdLogger has no field or method Output)
../../../../nlopes/slack/slack.go:110:9: logger.Output undefined (type stdLogger has no field or method Output)
# github.com/nlopes/slack
../../../../../../nlopes/slack/slack.go:104:9: logger.Output undefined (type stdLogger has no field or method Output)
../../../../../../nlopes/slack/slack.go:110:9: logger.Output undefined (type stdLogger has no field or method Output)
FAIL	github.com/mauleyzaola/kaizen/src/backend/tests/business [build failed]
[FAIL] One or more unit tests failed.
FAILURE
```

It looks the problem is with this commit: https://github.com/nlopes/slack/commit/28fbc53ee8cf0e0033c3852a222d47bc06b11b04

Someone forgot to add the `Output()` method 😄 

I can help if you like. Thanks!

--- a/slack.go
+++ b/slack.go
@@ -3,10 +3,10 @@ package slack
 import (
 	"context"
 	"errors"
+	"fmt"
 	"log"
 	"net/url"
 	"os"
-	"fmt"
 )
 
 var logger stdLogger // A logger that can be set by consumers
@@ -56,6 +56,8 @@ type stdLogger interface {
 	Panic(...interface{})
 	Panicf(string, ...interface{})
 	Panicln(...interface{})
+
+	Output(int, string) error
 }
 
 // SetLogger let's library users supply a logger, so that api debugging
@@ -101,7 +103,7 @@ func (api *Client) SetDebug(debug bool) {
 
 func (api *Client) Debugf(format string, v ...interface{}) {
 	if api.debug {
-		logger.Output(2,fmt.Sprintf(format, v...))
+		logger.Output(2, fmt.Sprintf(format, v...))
 	}
 }
 

